MineColab.ipynb

```diff
@@ -80,6 +73,71 @@
       "execution_count": null,
       "outputs": []
     },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_4IowSDVUZp8"
+      },
+      "source": [
+        "# Make-a-Server\n",
+        "\n",
+        "The code below will download a server for you and accept the EULA. After running these scripts, your server will be ready to run."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7wrNfinQOlWV"
+      },
+      "source": [
+        "**Download the Minecraft server**\n",
+        "\n",
+        "The code below will download Paper, a high-performance fork of the Vanilla server."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "Fy5-YjJMPV3S"
+      },
+      "source": [
+        "# Replace \"1.16.3\" with your desired server version.\n",
+        "# Available versions:\n",
+        "# - 1.16.3\n",
+        "# - 1.15.2\n",
+        "# - 1.14.4\n",
+        "# - 1.13.2\n",
+        "# - 1.12.2\n",
+        "# - 1.11.2\n",
+        "# - 1.10.2\n",
+        "# - 1.9.4\n",
+        "# - 1.8.8\n",
+        "# Newer versions might work too, however this isn't guaranteed.\n",
+        "version = '1.8.8'\n",
```

Comment:
Above in the comment its mentioned "Replace "1.16.3" with your desired server version."
However 1.8.8 is specified here as default.